packages/moonstone/Popup/Popup.js

```diff
@@ -575,7 +576,7 @@ class Popup extends React.Component {
 					onCloseButtonClick={onClose}
 					onHide={this.handlePopupHide}
 					onShow={this.handlePopupShow}
-					open={this.state.popupOpen >= OpenState.OPENING}
+					open={this.state.popupOpen > OpenState.CLOSED}
```

Comment:
This condition produces the same result so let's skip this change to keep it clear what was changed in this PR.

Reply:
@0x64 could you fix it for me?